cmake/EthUtils.cmake

```diff
@@ -62,3 +62,13 @@ macro(eth_add_test NAME)
 
 endmacro()
 
+# Creates C resources file from files
+function(eth_add_resources TARGET RESOURCE_FILE)
+
+	add_custom_target("${TARGET}.resources"
```

Comment:
I was trying to... the problem is that [add_custom_command](http://www.cmake.org/cmake/help/v3.0/command/add_custom_command.html) `PRE_BUILD` is not supported in every generator. And I need header files before I building the target.